docs/dqx/docs/demos.mdx

```diff
@@ -12,6 +12,9 @@ Import the following notebooks in the Databricks workspace to try DQX out:
 * [DQX Demo Notebook (library)](https://github.com/databrickslabs/dqx/blob/v0.7.0/demos/dqx_demo_library.py) - demonstrates how to use DQX as a library.
 * [DQX Lakeflow Pipelines Demo Notebook](https://github.com/databrickslabs/dqx/blob/v0.7.0/demos/dqx_dlt_demo.py) - demonstrates how to use DQX with Lakeflow Pipelines (formerly Delta Live Tables (DLT)).
 
+## Use with Databricks Asset Bundles
```

Comment:
Should this be moved under existing section `Use as a Library`. It does not seem to be a new category. Just different way of deployment when using dqx as a library.